selenium_tests/masquerade/canvas_base_page_object.py

```diff
@@ -0,0 +1,7 @@
+# from selenium_tests.base_page_object import BasePageObject
```

Comment:
(See comment about `class CanvasMasqueradePageObject` in [canvas_masquerade_page_object.py](https://github.com/Harvard-University-iCommons/lti_emailer/pull/133/files#r42396879)) -- if we're not using this file, it can be removed entirely; but if we're going to use it as a base class for the `CanvasMasqueradePageObject`, this code needs to be uncommented.